java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -29,6 +29,8 @@ public enum OrcConf {
   STRIPE_SIZE("orc.stripe.size", "hive.exec.orc.default.stripe.size",
       64L * 1024 * 1024,
       "Define the default ORC stripe size, in bytes."),
+  STRIPE_ROW_COUNT("orc.stripe.row.count","orc.stripe.row.count",
+      Integer.MAX_VALUE,"This value limit the row count in one stripe."),
```

Comment:
Keep a space after the comma.

Agree with setting the default value to Integer.MAX_VALUE, so that the default config is compatible with future user upgrades and has no additional impact.

Reply:
Please fix codestyle on line 32